doc/getting_started/beginner_tutorial.rst

```diff
@@ -25,163 +22,245 @@ Create a Repo
 
 .. code-block:: shell
+ 
+# on OSX
+$ pachctl get-file images master liberty.png | open -f -a /Applications/Preview.app
 
- # If we list the repos, we can see that there is now data
- $ pachctl list-repo
- NAME                CREATED             SIZE
- data                12 minutes ago       874 B
-
- # We can view the commit we just created
- pachctl list-commit data
- BRANCH              REPO/ID         PARENT              STARTED             FINISHED            SIZE
- master              data/master/0   <none>              6 minutes ago       6 minutes ago       874 B
-
- # We can also view the contents of the file that we just added
- $ pachctl get-file data master sales
- orange 	4
- banana 	2
- banana 	9
- orange 	9
+# on Linux
+ $ pachctl get-file images master liberty.png | display
  ...
 
 Create a Pipeline
 ^^^^^^^^^^^^^^^^^
 
-Now that we've got some data in our repo, it's time to do something with it.
-``Pipelines`` are the core primitive for Pachyderm's processing system (pps) and
-they're specified with a JSON encoding. For this example, we've already created the pipeline for you and it can be found at `examples/fruit_stand/pipeline.json on Github <https://github.com/pachyderm/pachyderm/blob/master/doc/examples/fruit_stand/pipeline.json>`_. Please open a new tab to view the pipeline while we talk through it.
+Now that we've got some data in our repo, it's time to do something with it. ``Pipelines`` are the core primitive for Pachyderm's processing system (pps) and they're specified with a JSON encoding. For this example, we've already created the pipeline for you and you can find the `code on Github <https://github.com/pachyderm/pachyderm/blob/master/doc/examples/opencv>`_. 
 
-When you want to create your own pipelines later, you can refer to the full :doc:`../deployment/pipeline_spec` to use more advanced options. This includes building your own code into a container instead of just using simple shell commands as we're doing here.
+When you want to create your own pipelines later, you can refer to the full :doc:`../reference/pipeline_spec` to use more advanced options. This includes building your own code into a container instead of the pre-built Docker image we'll be using here.
 
-For now, we're going to create a pipeline with 2 transformations in it. The first transformation filters the sales logs into separate records for apples,
-oranges and bananas. The second step sums these sales numbers into a final sales count.
+For now, we're going to create a single pipeline that takes in images and does some simple edge detection.
 
 .. code-block:: shell
 
- +----------+     +--------------+     +------------+
- |input data| --> |filter pipline| --> |sum pipeline|
- +----------+     +--------------+     +------------+
+ +----------+     +--------------+     +-------------+
+ |input data| --> |edge detection| --> | output data |
```

Comment:
Maybe use the image:

![opencv example](https://cloud.githubusercontent.com/assets/4524535/24355678/c9c9de8c-12c4-11e7-928f-e3f6c531c826.jpg)